images-list

```diff
@@ -137,6 +137,7 @@ library/nginx rancher/library-nginx 1.19.2-alpine
 library/nginx rancher/mirrored-library-nginx 1.19.2-alpine
 library/nginx rancher/mirrored-library-nginx 1.19.9-alpine
 library/traefik rancher/library-traefik 2.4.2
+library/traefik rancher/library-traefik 2.4.8
```

Comment:
Hey @erikwilson this works for now, but can you move this to a mirrored image ? By 2.6 we want to have everything that is mirrored and shipped with Rancher to be in that type of repo. So that would be: `rancher/mirrored-library-traefik`